content/posts/weekly/June-2023.md

```diff
@@ -8,3 +8,23 @@ weight: 51
 
 - Packit will now filter the pull requests based on the target branch if the `branch` is specified in the particular job configuration. ([packit-service#2074](https://github.com/packit/packit-service/pull/2074))
 - Packit now supports `manual_trigger` in the configuration that allows you to make jobs run only when explicitly triggered by a comment. ([packit-service#2069](https://github.com/packit/packit-service/pull/2069))
+
+## Week 23 (June 6th – June 12th)
+
+- Our dashboard now shows projects in a table instead of the cards.
+  ([dashboard#267](https://github.com/packit/dashboard/pull/267))
+- `/packit test` command has a new `--identifier` option. When this option is specified,
+  Testing Farm will trigger only jobs with that identifier.
+  ([packit-service#2079](https://github.com/packit/packit-service/pull/2079))
+- If the `vm_image_build job` doesn't have Copr project/owner configured,
+  Copr project used for the latest Copr build of the pull request will be now used.
+  ([packit-service#2080](https://github.com/packit/packit-service/pull/2080))
+- Fixed filters in `files_to_sync` not being applied properly.
+  ([packit#1977](https://github.com/packit/packit/pull/1977))
+- It is now possible to specify labels for a job and trigger tests based on labels
+  using a new `--labels` option of the `/packit test` command.
+  ([packit#1988](https://github.com/packit/packit/pull/1988),
+  [packit-service#2088](https://github.com/packit/packit-service/pull/2088))
+- We have finally fixed the issue that could've caused inconsistencies when tests
+  from a PR were supposed to be merged during the test runs on the Testing Farm.
+  ([packit-service#2090](https://github.com/packit/packit-service/pull/2090))
```

Comment:
@mfocko Do we wan to phrase this like that? There was no issue on "our" side..;)

```suggestion
- The issue that could've caused inconsistencies when tests
  from a PR were supposed to be merged during the test runs on the Testing Farm has been fixed.
  ([packit-service#2090](https://github.com/packit/packit-service/pull/2090))
```

or 

```suggestion
- The Testing Farm team have fixed the issue that could've caused inconsistencies when tests
  from a PR were supposed to be merged during the test runs on the Testing Farm.
  ([packit-service#2090](https://github.com/packit/packit-service/pull/2090))
```

:
:

Reply:
I guess the first suggestion makes more sense, no one to blame :wink: :smile: